### DISCUSSION TITLE
Documentation license / copyright

### DISCUSSION THREAD

**Discussion by peterhcharlton** [neuropsychology/NeuroKit#769] [General]
Neurokit is licensed under the highly permissive MIT license, whereas its documentation has what appears to be a highly restrictive copyright notice:

_By [Dominique Makowski](https://dominiquemakowski.github.io/) and the [Team](https://github.com/neuropsychology/NeuroKit/blob/master/AUTHORS.rst)
© Copyright 2020–2022._

Might you consider licensing the documentation under a more permissive license, e.g. CC BY?

Personally, this would be helpful as I would be keen to contribute to the documentation, but I would be hesitant if a copyright notice was going to be applied to my contribution, which I would interpret as preventing me from sharing the documentation elsewhere.

Thanks for considering this.

**Comment by DominiqueMakowski** [comment]
Interesting, I didn't know that. I am not good with licenses and stuff - I just went with filling in the default fields. Could you make a PR to change it to a better alternative?

**Comment by peterhcharlton** [comment]
I've made PR #771
